arm-resources/links/2016-09-01/swagger/links.json

```diff
@@ -0,0 +1,300 @@
+        "parameters": [
+          {
+            "name": "scope",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The fully qualified Id of the source resource scope. For example, to list resource links at and under a resource group, set the scope to /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myGroup.",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "enum": ["atScope()"],
+            "x-ms-enum": {
+              "name": "Filter",
+              "modelAsString": false
+            },
+            "type": "string",
+            "description": "The filter to apply on the list resource links operation. To list links only at the specified scope (not below the scope), use Filter.atScope()."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ResourceLinkResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "#/definitions/ResourceLinkFilter"
+      }
+    }
+  },
+  "definitions": {
+    "ResourceLinkFilter": {
+      "properties": {
+        "targetId": {
+          "type": "string",
+          "description": "The target Id of the resource."
+        }
+      },
+      "description": "Resource link filter."
+    },
+    "ResourceLinkResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ResourceLink"
+          },
+          "description": "The list of resource links."
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URL to get the next set of results."
+        }
+      },
+      "description": "List of resource links."
+    },
+    "ResourceLink": {
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The fully qualified Id of the resource link."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the resource link."
+        },
+        "properties": {
+          "$ref": "#/definitions/ResourceLinkProperties"
+        }
+      },
+      "description": "The resource link."
+    },
+    "ResourceLinkProperties": {
+      "properties": {
+        "sourceId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The fully qualified source resource Id. "
+        },
+        "targetId": {
```

Comment:
We use the properties for request body as well. Target Id is required in request body. Example here https://github.com/Azure/azure-sdk-for-net/pull/2462/files?diff=unified#diff-9b79bc6941ca9f099a07d8fdd87f70c2R67